CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -1554,3 +1641,28 @@ if IsNativePurchasing then
 		nativePurchaseFinished(wasPurchased)
 	end)
 end
+
+local function onVREnabled(vrEnabled) 
+	if vrEnabled then
+		IsVRMode = true
+		purchaseDialogVR:SetContent(PurchaseDialog)
+	else
+		IsVRMode = false
+		purchaseDialogVR:SetContent(nil)
+		PurchaseDialog.Parent = RobloxGui
+	end
+end
+
+spawn(function()
+	repeat wait() until Players.LocalPlayer ~= nil
+	onVREnabled(UserInputService.VREnabled)
```

Comment:
I'm not really sure, there might have been a reason at one point. Removing that.